_layouts/home.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: home
+---
+
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+{%- include head.html -%}
+
+  <body class="home">
+    <main class="home__content" aria-label="Content">
+        <div class="home__left">
+          <div class="home__left-inner">
+            <img class="home__title" src="{{relativeBase}}assets/images/lockbox-logo.svg" alt="{{ site.title | escape }}">
+            <h1>{{ content }}</h1>
+          </div>
+        <div class="home__app-buttons">
+          <a class="app-store" href="{{ site.ios_link | escape }}">
+            <img src="{{relativeBase}}assets/images/app-store.svg" alt="Apple App Store" height=" 50">
+          </a>
+          <a class="google-play-store" href="{{ site.android_link | escape }}">
+            <img src="{{relativeBase}}assets/images/gps.svg" height=" 50" alt="Google Play Store">
+          </a>
+        </div>
+        </div>
+        <div class="home__content-right">
+          <img src="./assets/images/devices.png">
```

Comment:
I built this locally and aXe says:

> Images must have alternate text

![Firefox_Lockbox___Firefox_Lockbox_project_documentation_and_website](https://user-images.githubusercontent.com/557895/54449560-a467f700-470b-11e9-85e5-469c0d705ee2.png)

Reply:
Added alt text, thanks @pdehaan 🙏